lib/async-await.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const fs = require('fs');
+const util = require('util');
+
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+
+/**
+ * @param  {} file
+ * @return stringData 
+ */
```

Comment:
Your JSDoc comments should have more description about what the function does

https://devhints.io/jsdoc